cranelift/codegen/src/isa/x64/abi.rs

```diff
@@ -480,9 +502,16 @@ impl ABIMachineSpec for X64ABIMachineSpec {
                     ));
                     cur_offset += 8;
                 }
-                // No XMM regs are callee-save, so we do not need to implement
-                // this.
-                _ => unimplemented!(),
+                RegClass::V128 => {
+                    cur_offset = (cur_offset + 15) & !15;
+                    insts.push(Inst::store(
+                        types::I8X16,
+                        r_reg.to_reg(),
+                        Amode::imm_reg(cur_offset, regs::rsp()),
+                    ));
```

Comment:
We don't have a "store on the stack" `Inst`?

Reply:
Switched to the usual `store` helper; not sure why the old code didn't use this.